src/SingleSelect.elm

```diff
@@ -305,34 +272,44 @@ optionId idx =
     "option-" ++ String.fromInt idx
 
 
-showOptions : Model msg a -> List ( Int, a ) -> Html msg
-showOptions model options =
-    if List.isEmpty options && model.searchText /= "" then
-        div [ class (classPrefix ++ "search-or-no-results-text") ] [ text ("No results found for: " ++ model.searchText) ]
+showOptions :
+    { internalMsg : Msg a -> msg
+    , options : List ( Int, a )
+    , optionType : String
```

Comment:
I don't like the name type here because you could also pass in any text. Maybe optionsDescription or just description or headline or I don't know :)